kasa/transports/sslaestransport.py

```diff
@@ -340,8 +379,51 @@ def generate_tag(request: str, local_nonce: str, pwd_hash: str, seq: int) -> str
 
     async def perform_handshake(self) -> None:
         """Perform the handshake."""
-        local_nonce, server_nonce, pwd_hash = await self.perform_handshake1()
-        await self.perform_handshake2(local_nonce, server_nonce, pwd_hash)
+        result = await self.perform_handshake1()
+        if result:
+            local_nonce, server_nonce, pwd_hash = result
+            await self.perform_handshake2(local_nonce, server_nonce, pwd_hash)
+
+    async def try_perform_less_secure_login(self, username: str, password: str) -> bool:
+        """Perform the md5 login."""
+        _LOGGER.debug("Performing less secure login ...")
+
+        pwd_hash = _md5_hash(password.encode())
+        body = {
+            "method": "login",
+            "params": {
+                "hashed": True,
+                "password": pwd_hash,
+                "username": username,
+            },
+        }
+
+        http_client = self._http_client
+        status_code, resp_dict = await http_client.post(
```

Comment:
```suggestion
        status_code, resp_dict = await self._http_client.post(
```
Unnecessary assignment.